src/tests/tcmalloc_unittest.cc

```diff
@@ -1290,6 +1290,7 @@ static int RunAllTests(int argc, char** argv) {
     free(p1);
     VerifyDeleteHookWasCalled();
 
+#if HAVE_DECL_CFREE || !defined(TCMALLOC_DISABLE_LIBC_OVERRIDE) || TCMALLOC_DISABLE_LIBC_OVERRIDE != 1
```

Comment:
Where does the new HAVE_DECL_CFREE come from?

Reply:
It existed before. But part of the override process (that this define disables) it gets defined if missing from the system (which it does on modern c libraries). Essentially trying to describe here when cfree is not available.